<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineLearning</class>
 <widget class="QWidget" name="MachineLearning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MachineLearning</string>
  </property>
  <widget class="QFrame" name="mlp_frame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>471</width>
     <height>531</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MLP Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mlp_plot"/>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="process_selection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Train Model Human Non Human</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Train Model Other Objects</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="parameter_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Echo size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sample_frequency">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2.86MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.14MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>114KHz</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QFrame" name="files_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_file">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_model">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="echo_file_name">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Model file name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_train">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Training</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="random_forest_frame">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>50</y>
     <width>481</width>
     <height>531</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Forest Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>461</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rf_plot"/>
   </widget>
  </widget>
  <widget class="QTableWidget" name="table_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>401</width>
     <height>16777215</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Label</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Train</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Test</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="machine_learning_method">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Train Size</string>
    </property>
   </widget>
   <widget class="QSlider" name="trainSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_train_size">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Train Size</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_data_open">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Processing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ml_prediction_open">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Machine Learning Prediction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ml_open">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Machine Learning</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
